docs/Guide.md

```diff
@@ -306,27 +324,28 @@ is dictated by the type of the attribute.
     })
 ```
 
-The example above produces response bodies of the form
-`[{"name"="foo"},{"name"="bar"}]` assuming the type `Account` only has a `name`
-attribute. The same example but with the line defining the response body
-(`Body("accounts")`) removed produces response bodies of the form:
-`{"accounts":[{"name"="foo"},{"name"="bar"}]` since `accounts` isn't used to
-define headers.
+The HTTP response body for requests sent to the "index" method are of the form
+`[{"name"="foo"},{"name"="bar"}]`. The same example but with the line defining
+the response body removed (`Body("accounts")`) produces response bodies of the
+form:
+`{"accounts":[{"name"="foo"},{"name"="bar"}]` since the shape of the response
```

Comment:
Was it meant to be `"name":"foo"` instead of `"name"="foo"`?

Reply:
yup, thank you! fixed.